.eslintrc.yaml

```diff
@@ -0,0 +1,3 @@
+# ---
+    parser: babel-eslint
+    
```

Comment:
Why do we need this new file?

Reply:
Eslint is being configured at the package.json level: https://github.com/cerner/terra-graphs/pull/5/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R27 

This should be removed / moved to there

Reply:
Updated here: aa1b4379362fbaee6235ee2d440d0dbb2a507fda
The extend line is commented out as it is causing several thousand linter errors. This will be resolved after the migration is complete:
https://github.com/cerner/terra-graphs/issues/8